projects/packages/search/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "jetpack-search",
+	"version": "0.2.0-alpha",
+	"description": "Package for Jetpack Search products",
+	"main": "main.js",
+	"directories": {
+		"test": "tests"
+	},
+	"scripts": {
+		"clean-all": "rm -rf build/* .cache/* node_modules/* node_modules/.[!.]* vendor/*",
```

Comment:
It seems strange to be removing all the files inside these directories without removing the directories themselves. Particularly for `node_modules` and `vendor` that seems likely to just confuse things.